gateway/src/main/java/io/zeebe/gateway/impl/job/ActivateJobsHandler.java

```diff
@@ -90,7 +90,7 @@ private void activateJobs(
                 response.getTruncated());
           },
           error -> {
-            Loggers.GATEWAY_LOGGER.warn(
+            Loggers.GATEWAY_LOGGER.trace(
```

Comment:
I think that's OK-ish as debug, no? Or is this happening too often with the rate limiter?

Reply:
The error code is not accessible at this point to do a conditional logging. So I would say, we leave it at level warn. In case of back pressure, gateway will be flooded with the log message. But let's assume that it is not a common case.

Reply:
:+1: